ui/app/contexts/metametrics.new.js

```diff
@@ -92,16 +92,25 @@ export function MetaMetricsProvider ({ children }) {
       const idTrait = metaMetricsId ? 'userId' : 'anonymousId'
       const idValue = metaMetricsId ?? METAMETRICS_ANONYMOUS_ID
       const match = matchPath(location.pathname, { path: PATHS_TO_CHECK, exact: true, strict: true })
-      if (
-        match &&
+      // Start by checking for a missing match route. If this falls through to the else if, then we know we
+      // have a matched route for tracking.
+      if (!match) {
+        // We have more specific pages for each type of transaction confirmation
+        // The user lands on /confirm-transaction first, then is redirected based on
+        // the contents of state.
+        if (location.pathname !== '/confirm-transaction') {
+          // Otherwise we are legitimately missing a matching route
+          // Ideally we'd include the details of match and previousMatch here, but breadcrumbs provides
+          // a fair bit of details that should be useful.
+          captureMessage(`Segment page tracking found unmatched route`, 'info')
```

Comment:
This message is much nicer! I was about to suggest moving the variable to the end, but relying on breadcrumbs works too!

Reply:
Though I did just open #9597 if we did want to update Sentry, and add the page as a tag or something

Reply:
Gonna wait for #9597 then add the location.pathname/previous match as extra details.. then I think adding a tag for the page would be cool. 

Perhaps even, as part of a separate PR, adding a tag when a match is found using the setTag API would add that tag to future events (i think). That could be a cool way to filter events by page/route (perhaps that can already be done)